imageresizerapp/imageresizerapp.nuspec

```diff
@@ -0,0 +1,24 @@
+<?xml version="1.0" encoding="utf-8"?>
+<package xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xmlns:xsd="http://www.w3.org/2001/XMLSchema">
+  <metadata>
+    <id>imageresizerapp</id>
+    <title>Image Resizer for Windows</title>
+    <version>3.0.4802</version>
```

Comment:
I was trying to go with just the major/minor, but I wanted to republish with the latest GitHub links. Unfortunately, no version change had occurred yet. I grabbed 4802 from the full version listed in the Windows Control Panel Programs and Features.

I really try to avoid using the timestamp versioning unless there's no other alternative (in this case, exposing a little bit of the internal version temporarily seemed ok).